assets/css/components/_c--section-nav.scss

```diff
@@ -5,24 +5,25 @@
 
 // component config
 //
-$section-nav-section-spacing: $spacing--2xl;
-$section-nav-sidebar-color:   $monochrome--grey;
-$section-nav-bp--sm:          $bp--sm;
-$section-nav-item-padding:    $spacing--xl;
+$section-nav-section-spacing-top:    $spacing--2xl;
+$section-nav-section-spacing-bottom: 4*$spacing--2xl;
```

Comment:
FYI: let's keep the multiplication, subtraction etc after the variable. I'm know it sounds picky, but the rest of the codebase does it that way and if we keep it consistent it's less to think about when looking at the code =).